### DISCUSSION TITLE
Preguntas acerca de como subir una pagina a GitHub Pages

### DISCUSSION THREAD

**Discussion by ByJ4Xx** [Launch-X-Latam/MisionFrontEnd#294] [Q&A]
Me encuentro subiendo mi practica de pastelería y luego de intentar subir la pagina como dicen varios videos no parece funcionar, mis preguntas son:
¿El archivo HTML tiene que llamarse "index.html"? En todos los videos y explicaciones he visto que tiene ese nombre.
¿Para subir otra pagina (en este caso la parte de admin de la pagina) se puede hacer desde el mismo repositorio o tengo que crear uno con las mismas características?

**Comment by DavidSantana872** [comment]
yo igual estuve intentando pero no se podia, https://youtu.be/EWm7AnFVSAU con este video si pude, creando la rama gh-page me funciono!
